pkg/autodiscovery/providers/kubelet.go

```diff
@@ -109,6 +122,17 @@ func parseKubeletPodlist(podlist []*kubelet.Pod) ([]integration.Config, error) {
 
 			configs = append(configs, c...)
 		}
+		adErrors = append(adErrors, utils.ValidateAnnotationsMatching(pod.Metadata.Annotations, containerIdentifiers, containerNames, adPrefix)...)
+		for _, err := range adErrors {
+			log.Errorf("Can't parse template for pod %s: %s", pod.Metadata.Name, err)
+			_, found := ADErrors[pod.Metadata.Name]
+			if !found {
+				ADErrors[pod.Metadata.Name] = map[string]bool{err.Error(): true}
+			} else {
+				ADErrors[pod.Metadata.Name][err.Error()] = true
```

Comment:
Let's use `ADErrors[pod.Metadata.Namespace/pod.Metadata.Name]` to avoid collision